docs/manual/cluster-admin/how-to-set-up-virtual-clusters.md

```diff
@@ -2,12 +2,14 @@
 
 ## What is Hived Scheduler and How to Configure it
 
-HiveD is a standalone component of OpenPAI, designed to be a Kubernetes Scheduler Extender for Multi-Tenant GPU clusters. A multi-tenant GPU cluster assumes multiple tenants (teams) share the same GPU pool in a single physical cluster (PC) and provides some resource guarantees to each tenant. HiveD models each tenant as a virtual cluster (VC), so that one tenant can use its own VC as if it is a private cluster, while it can also use other VCs' free resource at lower priority.
+HiveD is a standalone component of OpenPAI, designed to be a Kubernetes Scheduler Extender for Multi-Tenant GPU clusters. A multi-tenant GPU cluster assumes multiple tenants (teams) share the same GPU pool in a single physical cluster (PC) and provides some resource guarantees to each tenant. HiveD models each tenant as a virtual cluster (VC), so that one tenant can use its own VC as if it is a private cluster, while it can also use other VCs' free resource at lower priority. Hived supports both GPU virtual cluster and CPU-only cluster.
```

Comment:
Could we just "As a standalone component of OpenPAI, HiveD is a Kubernetes Scheduler for Deep Learning. For more please check [HiveD project](https://github.com/microsoft/hivedscheduler)"

Reply:
Changed the sentence to `As one standalone component of Microsoft OpenPAI, [HiveD](https://github.com/microsoft/hivedscheduler) is designed to be a Kubernetes Scheduler Extender for Multi-Tenant GPU clusters.`

Reply:
" Kubernetes Scheduler Extender" this can be changed in future.
And " GPU clusters" this is not exactly.
Still prefer "Kubernetes Scheduler for Deep Learning", it will not change and precise